application-chartjs-ui/src/main/resources/XWiki/ChartJS/TableToChartJS.xml

```diff
@@ -50,10 +50,15 @@
 |New Feature|5
 
 {{velocity}}
+  #set($chartOptions = {
```

Comment:
Inconsistent code style: here you don't leave a space between ``#set`` and ``(`` but below you do. I for one prefer to leave the space.

Reply:
Right, space added indeed (together with more options illustrating the new possibilities brought by the datalabels plugin).